lib/rust/ensogl/core/src/animation.rs

```diff
@@ -14,6 +14,8 @@ pub mod physics;
 mod frp;
 mod loops;
 
+#[allow(clippy::useless_attribute)]
+#[allow(ambiguous_glob_reexports)]
```

Comment:
Another case for `ambiguous_glob_reexports`. The thing is that both these modules reexport `Frp` structure, and new lint doesn't like it. I can't come up with a less traumatic way to address this. And also clippy is playing dumb and thinks that this attribute is useless (it's not true!).

Reply:
Please addd docs